Tools/cases_generator/eparser.py

```diff
@@ -0,0 +1,489 @@
+# C expression parser
```

Comment:
We'll need it as soon as we're going to generate super-instructions from basic opcodes, since stitching blocks of C code together is not _entirely_ trivial.